FRCSoftware2024.csv

```diff
@@ -4,31 +4,31 @@ NI-GameTools,ni-frc-2024-game-tools_24.0.1_offline.iso,https://download.ni.com/s
 NI-CompactRIO Driver,ni-compactrio-device-drivers_24.0.0.49260-0+f108_offline.iso,https://download.ni.com/support/nipkg/products/ni-c/ni-compactrio-device-drivers/24.0/offline/ni-compactrio-device-drivers_24.0.0.49260-0+f108_offline.iso,a76eef4d5d81d9f01a1c2ea860ba170a,FALSE
 NI-System Configuration,ni-system-configuration_24.0.0_offline.iso,https://download.ni.com/support/nipkg/products/ni-s/ni-system-configuration/24.0/offline/ni-system-configuration_24.0.0_offline.iso,952156ec99ed9b15fb38be981ee0600f,FALSE
 WPILibInstaller_Windows64,WPILib_Windows-2024.3.1.iso,https://packages.wpilib.workers.dev/installer/v2024.3.1/Win64/WPILib_Windows-2024.3.1.iso,781408b02146fde5a8275f31bd55a23c,FALSE
+WPILib VSCode Windows Archive,VSCode-1.85.1-Windows.zip,https://update.code.visualstudio.com/1.85.1/win32-x64-archive/stable,5621fc9203a0468f9270e2151ac2cdea,TRUE
```

Comment:
Is this not included anymore? WPILib fixed the VSCode install I thought?

Reply:
The PR that Ryan linked allows the WPILib installer to use a normal VsCode zip, so it enables this addition.